collectors/apps.plugin/README.md

```diff
@@ -64,7 +64,7 @@ Each of these sections provides the same number of charts:
     -   Major Page Faults (i.e. swap activity)
 -   Network
     -   Sockets Open
-
+    
```

Comment:
You removed any mentions of eBPF charts. We don't need the eBPF chart list here, but we need to mention that the eBPF plugin will show some charts here. I suggest to include a link to the eBPF chart list.

Reply:
Thank you for the suggestion, I brought the necessary paragraph.